docs/digital_analog_qc/analog-basics.md

```diff
@@ -110,19 +106,20 @@ from qadence import wait, chain, add
 
 pi = torch.pi
 
-# two qubit register
+# A two qubit register.
 reg = Register.from_coordinates([(0, 0), (0, 12)])
 
-# analog ansatz with input parameter
+# An analog ansatz with an input time parameter.
 t = FeatureParameter("t")
+
 block = chain(
-    AnalogRX(pi / 2),
+    AnalogRX(pi/2.),
     AnalogRZ(t),
     wait(1000 * VariationalParameter("theta", value=0.5)),
-    AnalogRX(pi / 2),
+    AnalogRX(pi/2),
```

Comment:
@Roland-djee use `2.` also here for consistency.

Reply:
Thanks.